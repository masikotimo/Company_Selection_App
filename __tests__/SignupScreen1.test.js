import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import configureMockStore from 'redux-mock-store';
import SignupScreen1 from '../src/screens/SignupScreen1';

const mockedStore = configureMockStore()({});
const testProps = createTestProps({});

describe('SignupScreen1', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SignupScreen1 {...testProps} store={mockedStore} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
