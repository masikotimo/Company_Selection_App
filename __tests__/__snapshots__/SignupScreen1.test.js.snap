// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignupScreen1 renders correctly 1`] = `
<View>
  <Button
    onPress={[Function]}
    title="gwwwweeee"
  />
</View>
`;
